server/index.js

```diff
@@ -1,16 +1,30 @@
 import path from "path";
 import express from "express";
+import expressSession from "express-session";
 import { userRouter } from "./routes/user.js";
 import { fileURLToPath } from "url";
 import { dirname } from "path";
+import dotenv from "dotenv";
 
+dotenv.config();
 const __filename = fileURLToPath(import.meta.url);
 const __dirname = dirname(__filename);
 
-const PORT = process.env.PORT || 3001;
+const PORT = 3001;
 
 const app = express();
 app.use(express.json());
+app.use(
+  expressSession({
+    name: "team-lp-project-5",
+    resave: false,
+    saveUninitialized: false,
+    secret: "aaa",
+    cookie: {
+      maxAge: 1000 * 60 * 60 * 8 * 365,
```

Comment:
That's a bit too long (you could even change it to 30s/60s for testing purposes) and if not, please add a comment next to it about what this calculation means.

Reply:
Done